pkg/agent/openflow/pipeline.go

```diff
@@ -894,14 +883,17 @@ func (c *client) l3FwdFlowToRemote(
 func (c *client) l3FwdFlowToRemoteViaGW(
 	localGatewayMAC net.HardwareAddr,
 	peerSubnet net.IPNet,
+	gwOFPort uint32,
 	category cookie.Category) binding.Flow {
 	ipProto := getIPProtocol(peerSubnet.IP)
 	l3FwdTable := c.pipeline[l3ForwardingTable]
 	return l3FwdTable.BuildFlow(priorityNormal).MatchProtocol(ipProto).
 		MatchDstIPNet(peerSubnet).
-		Action().DecTTL().
 		Action().SetDstMAC(localGatewayMAC).
-		Action().GotoTable(l3FwdTable.GetNext()).
```

Comment:
So previously the packet will go L2ForwardingCalcTable and Ingress tables?

Reply:
Correct. Originally, the packet to remote in NoEncap mode would go into L2ForwardingCalcTable and Ingress tables.